### DISCUSSION TITLE
how to use it with wmbusmeters

### DISCUSSION THREAD

**Discussion by gd35** [Man-fred/culfw.esp8266#14] [General]
I was able to compile the firmware and to upload it on my ESP8266. It seems to work fine since I was able to configure it to connect to my wlan. However, my plan is to use it with wmbusmeters to decode t1 frames coming from my water meter. Does the current version of the firmware can allow me to do it ? I have another question, it seems that one can interact with the device through MQTT but I do not see how to do it. Thanks in advance

**Comment by Man-fred** [comment]
Hello gd35, I don't know what protocol ist used by wmbusmeters. For my devices I have implemented SlowRf and MAX. So the original CUL has implemented WMBUS_T it shoud be possible. I will look witch functions are used and not implemented.
